Cassity_Zachary/Assignments/Great Number Game/greatnumber.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect,session
+import random
+app = Flask(__name__)
+app.secret_key = 'secret'
+
+too_low = 'Too Low!'
+too_high = 'Too High!'
+correct = 'Correct!'
+
+@app.route('/')
+def main_page():
+    if not 'rand' in session:
+        session['rand'] = random.randrange(0,101)
+    return render_template('great_number.html')
+@app.route('/test',methods = ['POST'])
+def guess():
+    if not 'guess' in session:
```

Comment:
no need for an if statement here.  you can just set the session['guess'] right away.